### PR TITLE
Add support for terraform 0.14.x versions.

### PR DESCRIPTION
**Pull request by brandocorp**
Attempting to add support for the newly release Terraform 0.14.0.

Fixes #428

### CODE REVIEW

**Comment by edwardbartholomew** [comment]
Thank you very much for your contribution @brandocorp 

Would you mind adding a test for 0.14 in Travis as you have time? For reference, here is test for 0.13 which could mostly be duplicated I believe:
https://github.com/newcontext-oss/kitchen-terraform/blob/master/.travis.yml#L119-L150

I'm sorry @KevinBuchs, more work for the GitHub Actions refactor.

**Comment by brandocorp** [comment]
@KevinBuchs, I sent a PR with a new test, but it is failing because the branch doesn't yet support 0.14.0 :grin: 

https://github.com/KevinBuchs/kitchen-terraform/pull/1/files

**Comment by wyardley** [comment]
Is it possible to get this out soon?

**Comment by aaron-lane** [comment]
@wyardley Thank you for your interest in the project! we're hoping to have it out tonight or tomorrow :crossed_fingers:

**Comment by wyardley** [comment]
It looks like we can set `verify_version: false` to downgrade to a warning; may look at doing that. Historically, we haven't seen too many problems when jumping versions.

**Comment by aaron-lane** [comment]
@wyardley yup, that's exactly why it's there! Given there aren't any logical gem changes in this branch, it seems like it should be safe to disable verification while we wrap this up.

**Comment by aaron-lane** [comment]
@brandocorp might you please rebase your branch against master? :pleading_face:

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased][unreleased]
 
+## [5.6.0] - 2020-12-04
```
Comment (aaron-lane) [inline CHANGELOG.md:11]: Let's update this date and get this released!

```suggestion
## [5.6.0] - 2020-12-08
```

**Review by aaron-lane — CHANGES_REQUESTED** [review]
Awesome possum 🤩 just need to update the change log date and we can release this

**Review by aaron-lane — APPROVED** [review]

**Pull request merged**